### DISCUSSION TITLE
New version?

### DISCUSSION THREAD

**Discussion by sfgvieira** [pivovarit/parallel-collectors#888] [General]
Any plans to release a new version? I was looking at [this](https://github.com/pivovarit/parallel-collectors/commit/a2c1bc2049ac596cdf1085054bef20f6246e1cb1) and would like to use it :D

**Comment by pivovarit** [comment]
Hopefully, next week :) I'm a bit swamped with @vavr-io https://x.com/pivovarit/status/1814901872005697824

**Comment by pivovarit** [comment]
I reviewed my plans for the future, and it turned out it's a good time to release 3.2.0 :) The artifacts are already uploaded to Maven Central - give them some time to propagate, though! Then I will publish an official release here
